testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/RegisterTest.java

```diff
@@ -460,7 +462,8 @@ public void registerUserUmlats() {
     // KEYCLOAK-3266
     @Test
     public void registerUserNotUsernamePasswordPolicy() {
-        adminClient.realm("test").update(RealmBuilder.create().passwordPolicy("notUsername").build());
+        adminClient.realm("test").update(RealmBuilder.create()
```

Comment:
Please remove this. Tests are by intent using a string and not a constant here.

Reply:
done